template/{{.input_root_dir}}/docs/mlops-setup.md.tmpl

```diff
@@ -77,6 +77,14 @@ and [Add a service principal to a workspace]({{ template `generate_doc_link` (ma
 for details.
 
 For your convenience, we also have a [Terraform module](https://registry.terraform.io/modules/databricks/mlops-aws-project/databricks/latest) that can set up your service principals.
+
+{{ else if eq .input_cloud `gcp` }}
```

Comment:
Looks identical to the code block above, let's replace ``` else if eq .input_cloud `aws` ``` with just `else` to capture both AWS and GCP

Reply:
Yeah so we can wrap that in an if-statement for AWS only